kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -4479,20 +4480,24 @@ public void onSuccess(Context context, KmApiResponse<KmFeedback> response) {
                             frameLayoutProgressbar.setVisibility(View.GONE);
                             if (response.getData() != null) { //i.e if feedback found
                                 //show the feedback based on the data given
-                                Message message = messageDatabaseService.getLatestStatusMessage(channel.getKey());
-
+                                Message latestStatusMessage = messageDatabaseService.getLatestStatusMessage(channel.getKey());
+                                Message latestMessage = messageList.get(messageList.size() - 1);
                                 //If Conversation was resolved and feedback was submitted given
```

Comment:
Fix this comment `If Conversation was resolved and feedback was submitted given`.